src/chrome/content/rules/Gittigidiyor.xml

```diff
@@ -18,28 +38,62 @@
 	<target host="gittigidiyor.com" />
 	<target host="www.gittigidiyor.com" />
 	<target host="aff.gittigidiyor.com" />
+	<target host="aff3.gittigidiyor.com" />
+	<target host="affiliate.gittigidiyor.com" />
+	<target host="afftrack.gittigidiyor.com" />
+	<target host="amp.gittigidiyor.com" />
+	<target host="antika.gittigidiyor.com" />
+	<target host="arama.gittigidiyor.com" />
+	<target host="bebek.gittigidiyor.com" />
+	<target host="bilgisayar.gittigidiyor.com" />
+	<target host="blog.gittigidiyor.com" />
+	<target host="cep-telefonu.gittigidiyor.com" />
 	<target host="cm.gittigidiyor.com" />
+	<target host="cms.gittigidiyor.com" />
+	<target host="cp.gittigidiyor.com" />
+	<target host="dekorasyon.gittigidiyor.com" />
 	<target host="dev.gittigidiyor.com" />
+	<target host="diger-hersey.gittigidiyor.com" />
 	<target host="dukkanlar.gittigidiyor.com" />
+	<target host="dvd.gittigidiyor.com" />
+	<target host="elektronik.gittigidiyor.com" />
+	<target host="emlak.gittigidiyor.com" />
+	<target host="ev-elektronigi.gittigidiyor.com" />
+	<target host="fotograf-kamera.gittigidiyor.com" />
+	<target host="gate.gittigidiyor.com" />
+	<target host="gate-ist.gittigidiyor.com" />
+	<target host="giyim.gittigidiyor.com" />
+	<target host="gunun-firsati.gittigidiyor.com" />
+	<target host="images.gittigidiyor.com" />
+	<target host="inceleme.gittigidiyor.com" />
+	<target host="kozmetik.gittigidiyor.com" />
+	<target host="kurumsal.gittigidiyor.com" />
 	<target host="m.gittigidiyor.com" />
+	<target host="mktg.gittigidiyor.com" />
+	<target host="mucevher-saat.gittigidiyor.com" />
+	<target host="muzik.gittigidiyor.com" />
+	<target host="oyuncak.gittigidiyor.com" />
+	<target host="lb.posta.gittigidiyor.com" />
 	<target host="profil.gittigidiyor.com" />
+	<target host="ses-goruntu.gittigidiyor.com" />
+	<target host="spor.gittigidiyor.com" />
 	<target host="urun.gittigidiyor.com" />
+	<target host="video-oyun.gittigidiyor.com" />
+	<target host="wap.gittigidiyor.com" />
+	<target host="yanit-merkezi.gittigidiyor.com" />
 	<target host="yardim.gittigidiyor.com" />
 
 	<target host="mcdn01.gittigidiyor.net" />
 		<test url="http://mcdn01.gittigidiyor.net/29801/tn24/298011980_tn24_0.jpg" />
 	<target host="mcdnaff03.gittigidiyor.net" />
-	<target host="mhd.gittigidiyor.net" />
 	<target host="st.gittigidiyor.net" />
 	<target host="st2.gittigidiyor.net" />
 
 	<!-- /$ doesn't work with HTTPS -->
-	<exclusion pattern="^http://mhd\.gittigidiyor\.net/$" />
```

Comment:
Please replace and re-add `mhd.gittigidiyor.net` with `mhd.gittigidiyor.com`

Reply:
Added, resolved conflict and rebased.